### PR TITLE
NAS-107723 / 20.10 / errno doesn't have EBADRPC attribute so use EHOSTUNREACH

### PR DESCRIPTION
**Pull request by yocalebo**

### CODE REVIEW

**Review by themylogin — COMMENTED** [review]
There are two EBADRPC usages in this file.

**Comment by yocalebo** [comment]
> There are two EBADRPC usages in this file.

good catch, fixed

**Review by themylogin — APPROVED** [review]

File: src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/remote.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def call(self, *args, **kwargs):
         try:
             if not self.connected.wait(timeout=20):
                 if self.remote_ip is None:
-                    raise CallError('Unable to determine remote node IP', errno.EBADRPC)
+                    raise CallError('Unable to determine remote node IP', errno.EHOSTUNREACH)
```
Comment (william-gr) [inline src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/remote.py:119]: Just making sure we dont look for that anywhere else in either middleware or UI?

I imagine we dont have that in Linux but we do in BSD?

Comment (yocalebo) [reply src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/remote.py:119]: @william-gr correct. Linux doesn't have EBADRPC, FreeBSD does. I didn't see any reference in middleware side but I'll verify webUI.

Comment (yocalebo) [reply src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/remote.py:119]: @william-gr verified that webUI doesn't reference it either.

Comment (william-gr) [reply src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/remote.py:119]: thanks for verifying!

**Pull request merged**